### PR TITLE
[rllib] Cleanups: remove special a3c ConvNet

### PR DESCRIPTION
**Pull request by ericl**

### CODE REVIEW

File: python/ray/rllib/models/convnet.py [hunk 1]
```diff
@@ -1,57 +0,0 @@
...
-                        initializer=initializer)
-    b = tf.get_variable(name + "/b", [size],
-                        initializer=tf.constant_initializer(bias_init))
-    return tf.matmul(x, w) + b
-
-
-class ConvolutionalNetwork(Model):
-    """Generic convolutional network."""
-
-    def _init(self, inputs, num_outputs, options):
```
Comment (ericl) [inline python/ray/rllib/models/convnet.py]: @richardliaw I presume this conv net gives higher performance with A3C on gym environments? If so, can we make it the standard model for all the algorithms?

Comment (richardliaw) [inline python/ray/rllib/models/convnet.py]: I haven't checked if it works well with DQN or the PG algorithms.

Comment (ericl) [inline python/ray/rllib/models/convnet.py]: I see, we should just check if this works ok then. Do you have a short running benchmark and a perhaps longer running (a couple hours) that I can try out?

Comment (richardliaw) [inline python/ray/rllib/models/convnet.py]: To use this model for the Policy Gradient and DQN code, code, you'd have to make a small adjustment such that `ModelCatalog.get_preprocessor` returns a filter that has an output shape of `42,42,1` - like this one https://github.com/ray-project/ray/blob/master/python/ray/rllib/a3c/envs.py#L24

Then you can run `train.py` on any ATARI domain... I don't think there's anything faster than `PongDeterministic-v4`.

Comment (ericl) [inline python/ray/rllib/models/convnet.py]: Ok, let's put this on hold until experiments are finished with 42x42x1 vs 80x80x3 on both PPO and this.

**Comment by richardliaw** [comment]
BTW I think most of these changes have been incorporated by #886

**Pull request closed without merging**